src/one_dragon/base/push/push_service.py

```
@@ -0,0 +1,162 @@
+        self.channels.append(channel)
+        self._id_2_channels[channel.channel_id] = channel
+        self._id_2_channel_schemas[channel.channel_id] = channel.config_schema
+
+    @cached_property
+    def push_config(self) -> PushConfig:
+        self.init_push_channels()
+        start_time = time.time()
+        while not self._inited:
+            time.sleep(1)
+            if time.time() - start_time > 5:
+                raise Exception('推送服务初始化超时')
```

Ruff (0.14.0)

Create your own exception

(TRY002)